chain/client/src/stateless_validation/chunk_endorsement_tracker.rs

```diff
@@ -33,15 +33,21 @@ impl ChunkEndorsementsState {
 
 /// Module to track chunk endorsements received from chunk validators.
 pub struct ChunkEndorsementTracker {
+    epoch_manager: Arc<dyn EpochManagerAdapter>,
+    inner: Mutex<ChunkEndorsementTrackerInner>,
+}
+
+struct ChunkEndorsementTrackerInner {
     epoch_manager: Arc<dyn EpochManagerAdapter>,
```

Comment:
can't we pass this epoch manager from the outer struct (line 36) when calling the methods of the inner?

Reply:
We could, but IMO it's a bit uglier than keeping two fields.